core/kits/assets/js/globals/global-select-behavior.js

```diff
@@ -0,0 +1,209 @@
+	// on elements that are not yet rendered when initialize() is called.
+	onRender() {
+		this.printGlobalSelectBox();
+
+		this.initGlobalPopover();
+
+		this.$el.addClass( 'elementor-control-global' );
+	}
+
+	toggleSelect() {
+		if ( this.popover.isVisible() ) {
+			this.popover.hide();
+		} else {
+			this.popover.show();
+		}
+	}
+
+	buildGlobalPopover() {
+		const $popover = jQuery( '<div>', { class: 'elementor-global-popover-container' } ),
+			$popoverTitle = jQuery( '<div>', { class: 'elementor-global-popover-title' } )
+				.html( '<i class="eicon-info-circle"></i>' + this.getOption( 'popoverTitle' ) ),
+			$manageGlobalPresetsLink = jQuery( '<div>', { class: 'elementor-global-manage-button' } )
+				.html( this.getOption( 'manageButtonText' ) + '<i class="eicon-cog"></i>' );
+
+		$popover.append( $popoverTitle, $manageGlobalPresetsLink );
+
+		return $popover;
+	}
+
+	printGlobalSelectBox() {
+		const $globalSelectBox = jQuery( '<div>', { class: 'elementor-global-select' } ),
+			$selectedGlobal = jQuery( '<span>', { class: 'elementor-global-selected' } )
+				.html( elementor.translate( 'default' ) );
+
+		$globalSelectBox.append( $selectedGlobal, '<i class="eicon-caret-down"></i>' );
+
+		this.$el.find( '.elementor-control-input-wrapper' ).prepend( $globalSelectBox );
+	}
+
+	initGlobalPopover() {
+		this.popover = elementorCommon.dialogsManager.createWidget( 'simple', {
+			className: 'elementor-global-control-popover',
+			message: this.buildGlobalPopover(),
+			effects: {
+				show: 'show',
+				hide: 'hide',
+			},
+			hide: {
+				onOutsideClick: false,
+			},
+			position: {
+				my: `center top`,
+				at: `center bottom+5`,
+				of: this.ui.controlContent,
+				autoRefresh: true,
+			},
+		} );
+
+		// Render the list of globals and append them to the Globals popover
+		this.view.getGlobalsList()
+			.then(
+			( globalsList ) => {
+				if ( globalsList ) {
+					this.addGlobalsListToPopover( globalsList );
+				} else {
+					this.showEmptyGlobalsMessage();
+				}
+			},
+			() => {
+				this.showEmptyGlobalsMessage();
+			} )
+			.finally( () => this.registerUiElementsAndEvents() );
+
+		this.createGlobalInfoTooltip();
+	}
+
+	showEmptyGlobalsMessage() {
+		const $message = jQuery( '<div>', { class: 'elementor-global-no-globals-found elementor-global-previews-container' } ).html( this.view.getNoGlobalsFoundMessage() );
+
+		this.popover.getElements( 'widget' ).find( '.elementor-global-popover-title' ).after( $message );
+	}
+
+	addGlobalsListToPopover( globalsList ) {
+		const $globalsList = this.view.buildGlobalsList( globalsList );
+
+		this.popover.getElements( 'widget' ).find( '.elementor-global-popover-title' ).after( $globalsList );
+	}
+
+	registerUiElementsAndEvents() {
+		// Instead of ui()
+		this.registerUiElements();
+
+		// Instead of events()
+		this.registerEvents();
+	}
+
+	// This method is not called directly, but triggered by Marionette's .triggerMethod()
```

Comment:
Where is this method called?

Reply:
In the controls (color and typography), in the `onAddGlobalButtonClick()` method